src/components/ble/AlertNotificationService.h

```diff
@@ -24,8 +27,25 @@ namespace Pinetime {
         int OnAlert(uint16_t conn_handle, uint16_t attr_handle,
                                     struct ble_gatt_access_ctxt *ctxt);
 
+        void event(char event);
+
+        static const char EVENT_HANG_UP_CALL = 0x00;
+        static const char EVENT_ANSWER_CALL = 0x01;
+
 
       private:
+        static const char ALERT_UNKNOWN = 0x01;
```

Comment:
these dont match the types defined in the spec, CALL should be 0x03 see https://developer.nordicsemi.com/nRF5_SDK/nRF51_SDK_v4.x.x/doc/html/group__ble__sdk__srv__ans__c.html

Reply:
Thanks! Pushing fix to this now